src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_create_and_update_with_managed_aad_enable_azure_rbac.yaml

```diff
@@ -0,0 +1,561 @@
+interactions:
+- request:
+    body: null
+    headers:
+      Accept:
+      - application/json
+      Accept-Encoding:
+      - gzip, deflate
+      CommandName:
+      - aks create
+      Connection:
+      - keep-alive
+      ParameterSetName:
+      - --resource-group --name --vm-set-type -c --enable-aad --enable-azure-rbac --aad-admin-group-object-ids
+      User-Agent:
+      - python/3.6.9 (Linux-4.19.104-microsoft-standard-x86_64-with-Ubuntu-18.04-bionic)
+        msrest/0.6.9 msrest_azure/0.6.3 azure-mgmt-resource/9.0.0 Azure-SDK-For-Python
+        AZURECLI/2.6.0
+      accept-language:
+      - en-US
+    method: GET
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourcegroups/clitest000001?api-version=2019-07-01
+  response:
+    body:
+      string: '{"id":"/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001","name":"clitest000001","type":"Microsoft.Resources/resourceGroups","location":"westus2","tags":{"product":"azurecli","cause":"automation","date":"2020-05-28T05:26:30Z"},"properties":{"provisioningState":"Succeeded"}}'
+    headers:
+      cache-control:
+      - no-cache
+      content-length:
+      - '313'
+      content-type:
+      - application/json; charset=utf-8
+      date:
+      - Thu, 28 May 2020 05:26:32 GMT
+      expires:
+      - '-1'
+      pragma:
+      - no-cache
+      strict-transport-security:
+      - max-age=31536000; includeSubDomains
+      vary:
+      - Accept-Encoding
+      x-content-type-options:
+      - nosniff
+    status:
+      code: 200
+      message: OK
+- request:
+    body: '{"location": "westus2", "properties": {"kubernetesVersion": "", "dnsPrefix":
+      "cliakstest-clitestvouk66jlj-c10894", "agentPoolProfiles": [{"count": 1, "vmSize":
+      "Standard_D2s_v3", "osType": "Linux", "type": "AvailabilitySet", "mode": "System",
+      "enableNodePublicIP": false, "scaleSetPriority": "Regular", "scaleSetEvictionPolicy":
+      "Delete", "name": "nodepool1"}], "linuxProfile": {"adminUsername": "azureuser",
+      "ssh": {"publicKeys": [{"keyData": "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDiOFgDVZSitF+jMxvBpJmYO8ReNVZMG+n996WvKxFfaR9903Zykxrvu7RUqdpZZ0hzzueeLJPJpiGxvk8ur4vnkXJkX7H8fvZZw6EI5PhuD76xXXRdXp9I2dH91oTqJ0Ne/s01K2PcJC+SzkU+icGLsIqP47QVoBL+H1nMapzcZVYYXzPd2u1bJZbGgJZSxEOJenybNY/1qhJGIEhMqLPXcz8QDuqLMrz3WMGLmfwv5SMTzd9ejfRAJ85RXeciTSjUtT37HUDafiql3JxhrrbUkgyuoVMztGp6DvTHb0XKDS06su1jBxQpGuquCKcpJNIa7Z3pJmWSM/b7VahsHzWP
+      pmiller@gmail.com\n"}]}}, "servicePrincipalProfile": {"clientId": "709e9be8-2b36-441f-81b5-16bd257fb5cb",
+      "secret": "b39ff07ddb2703825634$"}, "addonProfiles": {}, "enableRBAC": true,
```

Comment:
do we need to redact the credentials?

Reply:
we have the same in the other recordings

Reply:
they have been redacted.
https://github.com/Azure/azure-cli-extensions/tree/master/src/aks-preview/azext_aks_preview/tests/latest/recordings

Reply:
okay, will do